core/src/main/scala/filodb.core/query/ResultTypes.scala

```diff
@@ -22,9 +24,16 @@ final case class PartitionInfo(schema: RecordSchema, base: Array[Byte], offset:
 }
 
 /**
- * Describes column/field name and type
+ * Describes column/field name and type.
+ * isCumulative is not considered for equality/Hashcode.
  */
-final case class ColumnInfo(name: String, colType: Column.ColumnType, isCumulative: Boolean = true)
+final case class ColumnInfo(name: String, colType: Column.ColumnType, isCumulative: Boolean = true) {
+  override def equals(obj: Any): Boolean = obj match {
+    case ColumnInfo(n: String, ct: Column.ColumnType, _) => n == name && ct == colType
```

Comment:
singe its java String, lets user ``equals`` instead of ``==`` other wise lgtm

Reply:
Quick Question: Does java's String.equals work on null String objects ? In C#, String.equals failed on null strings ( NullReferenceException )

Reply:
(Super new to Java runtime, so still learning these things )

Reply:
@amolnayak311 it does invoke equals on String. unlike Java it doesn't do ref comparison.